http/src/main/resources/init-resources/msdsvmcontent/ms_dsvm_script_v5_cat.sh

```diff
@@ -15,11 +15,13 @@ apt-get install -y --no-install-recommends apt-utils
 SERVER_APP_PORT=8080
 SERVER_APP_TOKEN=''
 SERVER_APP_IP=''
+SERVER_APP_CERTFILE=''
+SERVER_APP_KEYFILE=''
 
 # Variables for listener
-#SERVER_APP_BASE_URL=''
-#SERVER_APP_ALLOW_ORIGIN='*'
-#SERVER_APP_WEBSOCKET_URL=''
+SERVER_APP_BASE_URL=/$2/
```

Comment:
can we declare all the script inputs in the beginning of the script and give them meaningful names

Reply:
Agree.